### DISCUSSION TITLE
Tracking more than 10 individuals

### DISCUSSION THREAD

**Discussion by TamaoMaeda** [talmolab/sleap#1667] [Help!] [answered]
I am currently analyzing a video that contains more than 30 individuals in a single frame.

SLEAP generally works well, but I encounter a limitation when correcting mistaken identities. The software only allows me to repair tracks 0 to 9. When I press the "control" button to correct a track, only tracks 0 to 9 appear on the screen, and I cannot select any track number greater than 10, as far as I understand.

Is there any other method to conduct proofreading for more than 10 individuals?

**Comment by Marielamend** [comment, marked as answer]
Hello @TamaoMaeda,

You are right, you can only select 0-9 with the control shortcut so here is a way to access the rest!
If you go to Tracks > Set Instance Track > New Track, you can add more tracks if needed to add more. As you are proofreading you can select the instance you want to correct and select the correct track under Set Instance Track.

![Screenshot 2024-01-22 155743](https://github.com/talmolab/sleap/assets/120222245/935ffe10-aa1e-4706-86b0-9ba4af7edf7c)
Rooting for you when conducting proofreading! Let us know if you have any other questions!

Have a great day,
Mariela

**Reply by TamaoMaeda** [reply]
Hello @Marielamend,
Thank you very much for the clear answer!
All the best,
Tamao
